### PR TITLE
[CLOB-947] New liquidity tier for TEST-USD

### PR DESCRIPTION
**Pull request by jonfung-dydx**

### CODE REVIEW

File: protocol/testing/genesis.sh [hunk 1]
```diff
@@ -135,6 +135,15 @@ function edit_genesis() {
...
 
+	# Liquidity Tier: For TEST-USD
+	dasel put -t json -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[]' -v "{}"
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].id' -v '4'
+	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].name' -v 'Nonlinear-trigger-easy'
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].initial_margin_ppm' -v '10007' # 1% + a little prime (100x leverage)
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].maintenance_fraction_ppm' -v '500009' # 50% of IM + a little prime
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].base_position_notional' -v '100000000' # 100 USDC
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].impact_notional' -v '50000000000' # 50_000 USDC (500 USDC / 1%)
+
```
Comment (jayy04) [inline protocol/testing/genesis.sh:146]: should we put this in `update_genesis_use_test_volatile_market`?

File: protocol/testing/genesis.sh [hunk 2]
```diff
@@ -135,6 +135,15 @@ function edit_genesis() {
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[3].base_position_notional' -v '1000000000' # 1_000 USDC
 	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[3].impact_notional' -v '2500000000' # 2_500 USDC (2_500 USDC / 100%)
 
+	# Liquidity Tier: For TEST-USD
+	dasel put -t json -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[]' -v "{}"
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].id' -v '4'
```
Comment (lucas-dydx) [inline protocol/testing/genesis.sh:140]: Does this overwrite `liquidity_tiers.[1]`? Shouldn't it be `liquidity_tiers.[4]`?

File: protocol/testing/genesis.sh [hunk 3]
```diff
@@ -135,6 +135,15 @@ function edit_genesis() {
...
+	dasel put -t string -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].name' -v 'Nonlinear-trigger-easy'
```
Comment (lucas-dydx) [inline protocol/testing/genesis.sh:141]: Nit: update this to something like `test-usd-100x-liq-tier`?

File: protocol/testing/genesis.sh [hunk 4]
```diff
@@ -135,6 +135,15 @@ function edit_genesis() {
...
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].initial_margin_ppm' -v '10007' # 1% + a little prime (100x leverage)
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].maintenance_fraction_ppm' -v '500009' # 50% of IM + a little prime
+	dasel put -t int -f "$GENESIS" '.app_state.perpetuals.liquidity_tiers.[1].base_position_notional' -v '100000000' # 100 USDC
```
Comment (lucas-dydx) [inline protocol/testing/genesis.sh:144]: Suggestion: let's make this number not round as well

File: protocol/testing/genesis.sh (hunk 1 above)
Comment (jonfung-dydx) [reply protocol/testing/genesis.sh:146]: was just gonna do that haha

**Review by jayy04 — APPROVED** [review]

**Pull request merged**